src/Tus/Client.php

```diff
@@ -454,6 +454,10 @@ protected function sendPatchRequest(int $bytes, int $offset) : int
                 throw new ConnectionException('Connection aborted by user.');
             }
 
+            if (HttpResponse::HTTP_UNSUPPORTED_MEDIA_TYPE === $statusCode) {
+                throw new Exception('Unsupported media Types.');
```

Comment:
I thought it was going to be changed to TusException?

Reply:
It is already from `Tus\Exception` namespace, we just need to rename it to `TusException` to avoid confusion